proposals/low-level-struct-improvements.md

```diff
@@ -486,13 +514,11 @@ The type will have the following definition:
 [AttributeUsage(AttributeTargets.Struct | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Parameter)]
 public sealed class LifetimeAnnotationAttribute : Attribute
 {
```

Comment:
@cston wanted to make sure you saw the change here. Think for a while we'll need to accetp both ctors since the attribute is out there but we ourselves should move to the single value case (assuming you agree with my design change here).

Reply:
We should consider a more general `int` value rather than one or two `bool` values, to allow for more expanded use in the future.

Reply:
I haven't had the chance to look closely here but I do think we should have some discussion for how the representation might evolve and how "older" and "newer" versions of the representation would coexist.

Reply:
This is why we changed the design from `bool` to `int`. That should allow us future expansion points into new lifetime areas that won't completely bust older compilers that encounter them.